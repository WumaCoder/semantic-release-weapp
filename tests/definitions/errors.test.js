const errors = require('../../src/definitions/errors');

describe('errors', () => {
  test('errors matches snapshot of errors', () => {
    expect(errors).toMatchSnapshot();
  });

  test('errors matches snapshot of errors', () => {
    expect(errors.EMISSINGfeishuWebhookUrl()).toMatchSnapshot();
  });
});
