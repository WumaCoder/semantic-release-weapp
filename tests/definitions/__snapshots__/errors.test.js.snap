// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`errors errors matches snapshot of errors 1`] = `
Object {
  "EMISSINGfeishuWebhookUrl": [Function],
}
`;

exports[`errors errors matches snapshot of errors 2`] = `
Object {
  "details": "A feishu webhook URL is required to post updates to a channel. Ensure you have the environment variable FEISHU_WEBHOOK_URL set appropriately and try again",
  "message": "Missing feishu Webhook Url",
}
`;
